GOVERNANCE.md

```diff
@@ -0,0 +1,192 @@
+correctness issues, interactions with other parts of the system, etc.
+
+**Defined by:** *approvers* entry in an OWNERS file in a repo owned by the
+Atlantis project.
+
+#### Requirements
+
+The following applies to the part of the codebase for which one would be an approver in an `OWNERS` file.
+
+- Member for at least 3 months
+- Author of at least 3 substantial PRs to the codebase, with the
+  definition of substantial subject to the lead's discretion (e.g.
+  refactors, enhancements rather than grammar correction or one-line pulls).
+- Exhibiting sound technical judgment through PR contributions
+- Exhibiting sound technical judgment through PR reviews
+- Sponsored by a Maintainer
+  - With no objections from other Maintainers or Steering Committee
+  - Done through PR to update the OWNERS file
+
+
+#### Responsibilities and privileges
+
+The following applies to the part of the codebase for which one would be a core-contributor
+in an `OWNERS` file.
+
+- Core-contributor status may be a precondition to accepting large code contributions
+- Demonstrate sound technical judgment
+- Responsible for project quality control via code reviews
+  - Focus on holistic acceptance of contributions such as dependencies with other features, backward / forwards
+    compatibility, API and flag definitions, etc
+- Expected to be responsive to review requests (inactivity may result in suspension until active again)
+- Mentor Members
+- May approve and merge code contributions for acceptance
+- Monitor Atlantis Slack (delayed response is perfectly acceptable), particularly for the area of your repository
+- Periodically attend the recurring community meetings
+- In general, continue to be willing to spend at least 5% of their time working on Atlantis (~0.25 business days per week)
+
+Each repository's current list of core-contributors is published and updated in each repo’s OWNERS.md file.
+
+### Maintainer
+
+Each repository in the Atlantis organization are allowed their own unique set of maintainers. Maintainers have the most experience with the given repo and are expected to have the knowledge and insight to lead its growth and improvement.
+
+New maintainers and subproject maintainers must be nominated by an existing maintainer and must be elected by a supermajority of existing maintainers. Likewise, maintainers can be removed by a supermajority of the existing maintainers or can resign by notifying one of the maintainers.
+
+If a Maintainer feels she/he can not fulfill the "Expectations from Maintainers", they are free to step down.
+
+In general, adding and removing maintainers for a given repo is the responsibility of the existing maintainer team for that repo and therefore does not require approval from the steering committee. However, in rare cases, the steering committee can veto the addition of a new maintainer by following the conflict resolution process.
+
+Responsibilities include:
+
+- Strong commitment to the project
```

Comment:
I like this, but how is that measured; its a bit more subjective than the other responsibilities listed.